cloudformation/cfncluster.cfn.json

```diff
@@ -734,21 +735,14 @@
     },
     "VolumeSize": {
       "Description": "Size of EBS volume in GB, if creating a new one",
-      "Type": "Number",
-      "Default": "20"
+      "Type": "String",
+      "Default": "20, 20, 20, 20, 20"
     },
     "VolumeType": {
       "Description": "Type of volume to create either new or from snapshot",
```

Comment:
This should be "Comma delimited list of ..."

Reply:
Apparently the Cloud Formation Template does not allow passing comma separated list parameters into a substack. When I changed the type from string to comma separated list the template produced an error.